selfdrive/locationd/laikad.py

```diff
@@ -20,15 +21,31 @@
 from system.swaglog import cloudlog
 
 MAX_TIME_GAP = 10
+EPHEMERIS_CACHE = 'LaikadEphemeris'
 
 
 class Laikad:
-
   def __init__(self, valid_const=("GPS", "GLONASS"), auto_update=False, valid_ephem_types=(EphemerisType.ULTRA_RAPID_ORBIT, EphemerisType.NAV)):
-    self.astro_dog = AstroDog(valid_const=valid_const, auto_update=auto_update, valid_ephem_types=valid_ephem_types)
+    self.astro_dog = AstroDog(valid_const=valid_const, auto_update=auto_update, valid_ephem_types=valid_ephem_types, clear_old_ephemeris=True)
     self.gnss_kf = GNSSKalman(GENERATED_DIR)
     self.orbit_p: Optional[Process] = None
     self.orbit_q = Queue()
+    self.load_cache()
+    self.last_save_t = None
+    self.last_fetch_orbits_t = None
+
+  def load_cache(self):
+    params = Params()
+    if params.check_key(EPHEMERIS_CACHE) and False: # todo fix get params
```

Comment:
`Params::checkKey` only tells you whether this is a valid params key. it doesn't really make sense to use it here

Reply:
Thanks! Will remove it